### PR TITLE
[Workers] Environments revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-6577

### CODE REVIEW

File: content/workers/wrangler/environments.md [hunk 1]
```diff
@@ -163,7 +114,7 @@ if (ENVIRONMENT === "staging") {
 }
 ```
 
-### Staging Environment with workers.dev
+### Staging environment with \*.workers.dev
 
 In order to deploy your code to your `*.workers.dev` subdomain, include `workers_dev = true` in the desired environment. Your `wrangler.toml` file may look like this:
```
Comment (pedrosousa) [inline content/workers/wrangler/environments.md]:
```suggestion
To deploy your code to your `*.workers.dev` subdomain, include `workers_dev = true` in the desired environment. Your `wrangler.toml` file may look like this:
```

File: content/workers/wrangler/environments.md [hunk 2]
```diff
@@ -7,118 +7,69 @@ title: Environments
...
 
----
-
-## Examples
-
-### Top-level configuration
+You cannot specify multiple environments with the same name.
 
-#### Publishing to a custom domain
+Wrangler appends the environment name to the top-level name to deploy a Worker. For example, a Worker project named `my-worker` with an environment `[env.dev]` would deploy a Worker named `my-worker-dev`.
```
Comment (pedrosousa) [inline content/workers/wrangler/environments.md:50]: There are a few relevant notes here at the end of the configuration section, but after reading this part I still had the following questions:
- Do environment-specific configurations override "global" configurations in the same file? I believe this is shown in the first example, but it's not explicitly said in the text.
- Do we need to put all "global" settings before env-specific settings? If it's not required, how do we end an env-specific section? With an empty line? [This is actually related to the TOML file format itself — I don't know if we briefly explain it somewhere else.]
- _(Related to the previous question — it only makes sense if we can add global settings anywhere on the file.)_ Is the order important, i.e., if we add environment-specific configuration first and then add global configuration settings, will the env-specific settings still apply?

File: content/workers/wrangler/environments.md [hunk 3]
```diff
@@ -181,61 +132,22 @@ workers_dev = true
...
-~/my-worker $ wrangler publish --env production
-Uploaded my-worker
-Published my-worker
-  https://my-worker.<your-subdomain>.workers.dev
-```
-
-{{<Aside type="note">}}
-
-When you create a Service or Environment, Cloudflare automatically registers an SSL certification for it. SSL certifications are discoverable and a matter of public record. Be careful when naming your Services and Environments that they do not contain sensitive information i.e `migrating-service-from-company1-to-company2` or `company1-acquisition-load-test`.
+When you create a Worker or environment, Cloudflare automatically registers an SSL certification for it. SSL certifications are discoverable and a matter of public record. Be careful when naming your Services and Environments that they do not contain sensitive information, such as, `migrating-service-from-company1-to-company2` or `company1-acquisition-load-test`.
```
Comment (pedrosousa) [inline content/workers/wrangler/environments.md]: Nit: Not sure if `registers an SSL certification` is clear... Do we create a new SSL/TLS certificate? Do we update an existing SSL/TLS certificate? Can we link to a page with more information on this behavior?

Also, the last sentence doesn't seem related to the first one... If that's really the case, I would create a separate paragraph for it.

**Review by pedrosousa — APPROVED** [review]

File: content/workers/wrangler/environments.md [hunk 4]
```diff
@@ -7,118 +7,69 @@ title: Environments
...
-### Naming
+## Configuration
 
-You cannot specify multiple environments with the same name. If this were allowed, publishing each environment would overwrite your previously deployed Worker and the behavior would not be clear.
+To configure an environment:
 
-For this reason, Wrangler appends the environment name to the top-level name to publish a Worker. For example, a Worker project named `my-worker` with an environment `[env.dev]` would become `my-worker-dev`.
+1. Open your Worker's `wrangler.toml` file.
+2. Add `[env.<NAME>]` and change `<NAME>` to the desired name of your environment.
+3. Repeat step 2 to create multiple environments.
```
Comment (deadlypants1973) [inline content/workers/wrangler/environments.md:31]:
```suggestion
3. Repeat step 2 to create multiple environments.

Be careful when naming your environments that they do not contain sensitive information, such as, `migrating-service-from-company1-to-company2` or `company1-acquisition-load-test`.
```

File: content/workers/wrangler/environments.md (hunk 2 above)
Comment (pedrosousa) [reply content/workers/wrangler/environments.md:50]: I found relevant information about environments here: 
https://kate-fixes-env-revision.cloudflare-docs-7ou.pages.dev/workers/wrangler/configuration/#environments

We should probably link to this section, or move some of the content to the "Environments" page. What do you think?

Comment (deadlypants1973) [reply content/workers/wrangler/environments.md:50]: Will add a link! @pedrosousa

File: content/workers/wrangler/environments.md [hunk 5]
```diff
@@ -181,61 +134,22 @@ workers_dev = true
...
-~/my-worker $ wrangler publish --env production
-Uploaded my-worker
-Published my-worker
-  https://my-worker.<your-subdomain>.workers.dev
-```
-
-{{<Aside type="note">}}
-
-When you create a Service or Environment, Cloudflare automatically registers an SSL certification for it. SSL certifications are discoverable and a matter of public record. Be careful when naming your Services and Environments that they do not contain sensitive information i.e `migrating-service-from-company1-to-company2` or `company1-acquisition-load-test`.
+When you create a Worker or environment, Cloudflare automatically registers an SSL certification for it. SSL certifications are discoverable and a matter of public record. Be careful when naming your Services and Environments that they do not contain sensitive information, such as, `migrating-service-from-company1-to-company2` or `company1-acquisition-load-test`.
```
Comment (deadlypants1973) [inline content/workers/wrangler/environments.md]:
```suggestion
When you create a Worker via an environment, Cloudflare automatically creates an SSL certification for it. SSL certifications are discoverable and a matter of public record.
```
